### PR TITLE
add log entry if proxies are being used

### PR DESCRIPTION
**Pull request by mathiask88**
https://github.com/prebuild/prebuild-install/issues/51

@ralphtheninja Something like this?

### CODE REVIEW

**Comment by ralphtheninja** [comment]
Exactly like that! :)

File: download.js [hunk 1]
```diff
@@ -64,6 +64,10 @@ function downloadPrebuild (opts, cb) {
               proxyAuth: parsedProxy.auth
             }
           })
+          log.http('request', 'Proxy setup detected (Host: ' +
+            parsedProxy.hostname + ', Port: ' +
+            parsedProxy.port + ', Authentication: ' +
+            (parsedProxy.auth ? 'Yes' : 'No') + ')')
```
Comment (ralphtheninja) [inline download.js]: Maybe add `tunnelFnName` as well?

Comment (mathiask88) [reply download.js]: Sure, why not.

**Comment by ralphtheninja** [comment]
2.4.1

**Pull request merged**